src/PendingMove.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Scanner;
+
+public class PendingMove {
+    private Game game;
+    private Boolean validMove;
+
+    PendingMove(Game game) {
+        this.game = game;
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Please enter a X coordinate.");
+        String input = scanner.nextLine();
+        int x = Integer.parseInt(input.trim());
+        System.out.println("Please enter a Y coordinate.");
+        input = scanner.nextLine();
+        int y = Integer.parseInt(input.trim());
+
+        Move move = getMove(game.getCurrentPlayer().pawnToMove(), new Point(x,y));
```

Comment:
If we think of this class like a form, this code also naturally becomes a **query** method like `getValidMove`. Any time the input changes, we would want `getValidMove` to return the current valid move.